pyproject.toml

```diff
@@ -1,3 +1,6 @@
 [build-system]
 requires = ["setuptools>=42"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+markers = "ignore_cloud_tests: Mark tests to be ignored when running with env set to cloud_tests"
```

Comment:
nit: this marker has been confusing me for quite a while.  A better name for it is probably `ignore_during_cloud_tests`